i18n/shared/src/main/scala/io/udash/i18n/RemoteTranslationRPC.scala

```diff
@@ -21,5 +22,4 @@ trait RemoteTranslationRPC {
   /** Returns map of translations and bundle hash. If `oldHash` is not outdated, this Future will contain None. */
   def loadTranslationsForLang(lang: Lang, oldHash: BundleHash): Future[Option[Bundle]]
 }
-
-
+object RemoteTranslationRPC extends RPCTypeClasses[DefaultServerUdashRPCFramework.type, RemoteTranslationRPC]
```

Comment:
This by no means makes any assumption about usage of `DefaultServerUdashRPCFramework`. It simply adds instances for it since it's the framework most likely to be used.